### PR TITLE
Initial OSSF repository docs

### PR DESCRIPTION
**Pull request by balteravishay**
This PR proposes an initial readme and charter docs, as well as linting GH actions and settings yaml file for the memory safety SIG, forming under the OSSF BEST WG.

### CODE REVIEW

**Review by ljharb — CHANGES_REQUESTED** [review]

File: .github/settings.yml [hunk 1]
```diff
@@ -17,13 +17,15 @@ repository:
...
-
-  - username: 
+  - username: walterpearce
+    permission: admin
+  - username: SecurityCRob
+    permission: admin
+  - username: david-a-wheeler
+    permission: admin
+  - username: balteravishay
     permission: push
```
[the remark this answers is not in the document]
Comment (balteravishay) [reply .github/settings.yml:29]: thanks for the review!
the intention was not to provide access rights to myself where i aught not have them, but to follow how other repositories in the OSSF org have implemented the settings file, for instance the [vulnerability disclosures WG](https://github.com/ossf/wg-vulnerability-disclosures/blob/main/.github/settings.yml), the [supply chain integrity WG](https://github.com/ossf/wg-supply-chain-integrity/blob/main/.github/settings.yml).
Should I leave the collaborators part empty, as the template was, if that is the intention?

[the remark this answers is not in the document]
Comment (ljharb) [reply .github/settings.yml:29]: Yes, let’s leave it empty for now.

**Review by SecurityCRob — APPROVED** [review]

File: .github/settings.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (balteravishay) [reply .github/settings.yml]: fixed

**Review by SecurityCRob — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,82 +1,83 @@
...
 
+## **Motivation**
 
-## 
-**Motivation**
+Memory safety vulnerabilities, caused by mistakes in memory management, are common in unsafe programming languages like C and C++.This type of vulnerability is responsible for a majority of security breaches, with estimates from Microsoft and Google showing that up to 70% and 90% of vulnerabilities in their products, respectively, are memory safety vulnerabilities. 
 
-[Background / use cases of the problem to be solved]
+Safe languages like Rust, Go, and Java are less prone to these types of errors. The consequences of these vulnerabilities are not just technical, but can result in significant financial losses and invasion of personal data and privacy. A recent analysis by Google Project Zero showed that 67% of vulnerabilities exploited in the wild were due to a lack of memory safety, making it a critical issue that needs to be addressed in software development.
```
[the remark this answers is not in the document]
Comment (balteravishay) [reply README.md]: fixed

**Review by nellshamrell — APPROVED** [review]

**Pull request merged**